cameras/README.md

```diff
@@ -0,0 +1,52 @@
+# rtsp
+This repo contains RTSP url paths for cameras and NVRs. 
+
+The OUI (*O*rganization *U*nique *I*dentifier) of the MAC address 
+is used to help narrow the list of candidate camera/NVR models when
+scanning the local network for devices. Even though multiple manufacturers
+may share the same OUI, because they use the same supplier of network
+interface cards/chipsets, the use of the OUI makes scanning faster.
+
+
+## Fields
+| Field | Description | Required |
+| ----- | ----------- | -------- |
+| company	| The name of the camera manufacturer, e.g. "Sharx" or "Amcrest" | required |
+| type | `camera` for standalone IP cameras, or `nvr` for Network Video Recorders | required |
+model	| The model name | required |
+oui_regex	| The regular expression that matches the OUI of the MAC Address. It's OK to assume uppercase only with colon separators like `(00:80:F0)` | required |
+rtsp_url	| The full url that reaches the H.264 RTSP video stream | required |
+stream_name	| A `;` separated list of user-facing description of each stream, e.g `high-res 1080p; low-res 640p`. Defaults to `Stream #{{stream}}` | optional |
```

Comment:
`stream_names` and `channel_names` just for symmetry with `streams` and `channels`